bin/plotting/pycbc_plot_bank_corner

```diff
@@ -0,0 +1,282 @@
+parser.add_argument("--output-plot-file",
+    required=True,
+    help="Name of the output file")
+parser.add_argument("--parameters",
+    nargs="+",
+    default=[],
+    action=option_utils.ParseParametersArg,
+    metavar="PARAM[:LABEL]",
+    help="Only plot the given parameters. May also provide a "
+         "label for each parameter. Choose from a combination of " + \
+         ", ".join(conversion_options)  + ", though some "
+         "may not be buildable from bank parameters. Can optionally "
+         "also specify a LABEL for each parameter. If no LABEL is "
+         "provided, PARAM will used as the LABEL. If LABEL "
+         "is the same as a parameter in "
+         "pycbc.waveform.parameters, the label "
+         "property of that parameter will be used. If not "
+         "provided, will plot all of the parameters in the "
+         "bank.")
+parser.add_argument("--log-parameters",
+    nargs="+",
+    default=[],
+    help="Plot these parameters on a log scale")
+parser.add_argument('--plot-histogram',
+    action='store_true',
+    help="Plot 1D histograms of parameters on the "
+         "diagonal axes.")
+# add mins, maxs options
+parser.add_argument('--mins',
+    nargs='+',
+    metavar='PARAM:VAL',
+    default=[],
+    help="Specify minimum parameter values to plot. This "
+         "should be done by specifying the parameter name "
+         "followed by the value. Parameter names must be "
+         "the same as the PARAM argument in --parameters "
+         "(or, if no parameters are provided, the same as "
+         "the parameter name specified in the variable "
+         "args in the input bank. If none provided, "
+         "the smallest parameter value in the posterior "
+         "will be used.")
+parser.add_argument('--maxs',
+    nargs='+',
+    metavar='PARAM:VAL',
+    default=[],
+    help="Same as mins, but for the maximum values to "
+         "plot.")
+parser.add_argument("--color-parameter",
+    nargs=1,
+    action=option_utils.ParseParametersArg,
+    metavar='PARAM[:LABEL]',
+    help="Color scatter points according to the parameter given. "
+         "May optionally provide a label in the same way as for "
+         "--parameters. Default=No scatter point coloring.")
+parser.add_argument('--dpi',
+    type=int,
+    default=200,
+    help="Set the DPI of the plot. Default is 200.")
+add_style_opt_to_parser(parser)
+args = parser.parse_args()
+
+init_logging(args.verbose)
+set_style_from_cli(args)
+
+mins, maxs = option_utils.plot_ranges_from_cli(args)
+
+logging.info("Reading in the bank")
+bank = {}
+with h5py.File(args.bank_file, 'r') as bankf:
+    for p in bankf.keys():
+        if not isinstance(bankf[p], h5py.Dataset): continue
+        # Ignore things which aren't numbers, which cannot
+        # be histogrammed - e.g. approximant
+        if not np.issubdtype(bankf[p].dtype, np.number): continue
+        bank[p] = bankf[p][:]
+
+banklen = list(bank.values())[0].size
```

Comment:
This may require more RAM than is reasonable for such an operation. Can you instead set `banklen` in the `for` loop above like `banklen = len(bankf[p])`?